src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -123,6 +133,23 @@
         return fileInfos;
     }
 
+    /**
+     * Returns the list of non-binary files for the repo {@code repoConfig}.
+     */
+    public static Set<Path> getListOfNonBinaryFiles(RepoConfiguration repoConfig) {
+        String receivedMsg = GitDiff.gitGetModifiedFiles(Paths.get(repoConfig.getRepoRoot()));
+        String[] listOfFiles = receivedMsg.split("\n");
+
+        // Gets rid of binary files and files with invalid directory name, then convert into type path
+        // and return as a Set.
```

Comment:
I think `then convert into type path and return as a Set.` is not necessary as the message is easily implied from the code.